lua/rest-nvim/init.lua

```diff
@@ -25,8 +25,15 @@ rest.run = function(verbose)
     raw = config.skip_ssl_verification and { "-k" } or nil,
     body = result.body,
     dry_run = verbose or false,
+    bufnr = result.bufnr,
+    start_line = result.start_line,
+    end_line = result.end_line,
   }
 
+  if config.get("highlight").enabled == true then
```

Comment:
We can skip the `== true` here, it's not really needed :p

Reply:
ok